docs/griptape-framework/misc/events.md

```diff
@@ -131,24 +131,26 @@ Handler 2 <class 'griptape.events.finish_structure_run_event.FinishStructureRunE
 You can use the [CompletionChunkEvent](../../reference/griptape/events/completion_chunk_event.md) to stream the completion results from Prompt Drivers.
 
 ```python
-from griptape.events import CompletionChunkEvent, EventListener
+from griptape.events import CompletionChunkEvent, EventListener, EventBus
 from griptape.tasks import ToolkitTask
 from griptape.structures import Pipeline
 from griptape.tools import WebScraper, TaskMemoryClient
 from griptape.config import OpenAiStructureConfig
 from griptape.drivers import OpenAiChatPromptDriver
 
 
+
+EventBus.event_listeners = [
```

Comment:
I think I'd prefer `EventBus.add_event_listener()` similar to `EventBus.publish_event()`

Reply:
Oh, now I realize that there is a method for this. Can we prefer that one in examples?

Reply:
Yep updated 👍